server/schedulers/adjacent_region.go

```diff
@@ -46,16 +46,16 @@ func init() {
 		return func(v interface{}) error {
 			conf, ok := v.(*balanceAdjacentRegionConfig)
 			if !ok {
-				return ErrScheduleConfigNotExist
+				return errs.ErrScheduleConfigNotExist
 			}
 			if len(args) == 2 {
 				leaderLimit, err := strconv.ParseUint(args[0], 10, 64)
 				if err != nil {
-					return errors.WithStack(err)
+					return errs.ErrSchedulerConfig.FastGenByArgs("leader limit")
```

Comment:
cc @rleungx 
how about adding a unify comment about this situation, we miss the error details info for this situation.

Reply:
I think we only need to know which config cannot be parsed is enough. Or I will add its input value later.

Reply:
I think we only need to know which config cannot be parsed is enough. Or I will add its input value later.